### ISSUE TITLE
vt_kable_* is hardcoded to be latex

### ISSUE THREAD

**Issue by thebioengineer** [phuse-org/valtools#101]
for almost all `vt_kable_*` functions, the format argument is set to latex by default. Can we get away from setting it and just allow user to set via ... to kable? or not set at all?

```
vt_kable_* <- function(data, ...){
   ## function guts
   kable(data, ...)
}

**Comment by mariev** [comment]
I think the original intent was that minimal call to  `vt_kable*` would give the object that SCHARP uses.  I think we can get around this if we include ` knitr::opts_knit$set(out.format = "latex")` in setup chunk for all templates

**Comment by thebioengineer** [comment]
if we render to pdf, kable would automatically do this though - is there a specific need to define the format?

**Comment by mariev** [comment]
addressed via #110 & #115
